src/audio/component.c

```diff
@@ -47,7 +47,7 @@ struct comp_data {
 
 static struct comp_data *cd;
 
-static struct comp_driver *get_drv(uint32_t type)
+static struct comp_driver *get_drv(uint32_t type, uint16_t flavour)
```

Comment:
I assume a *flavour* will be sof_mux_flavour ? If so does it mean every component will be *MUX* like?

Reply:
No it means we can have several diferent flavours of common components like mixer, mux, volume whereby we can pick the correct version that matches the use case. The driver wont care about the flavour of mux or mixer etc, it will treat it as a normal mux, muxer etc.